PROJECT_CHARTER.md

```diff
@@ -0,0 +1,72 @@
+
+## Section 1: Scope
+
+The project tries to provide users a variety of building blocks to create a highly customizable, maintainable and scaleable framework to build and run end-to-end (e2e) tests for web and mobile applications. While it is focused to run these tests based on the [WebDriver](https://w3c.github.io/webdriver/) standard it leverages also other interfaces to provide as much control of the automated user agent as possible.
+
+With that the project offers not only the ability to automate user agents using open standards, it also enforces a set of common good practices that help to avoid pitfalls as well as the ability to connect predefined solutions to problem spaces that were already solved by the WebdriverIO community.
+
+### 1.1: In-scope
+
+- automation of modern web browser (e.g. Chrome, Firefox, Safari, Edge)
+- automation of mobile devices (web as well as native applications)
+- testing applications from end-to-end at scale (including parallisation and reporting)
+- integration into 3rd party services and vendors (e.g. [Sauce Labs](https://saucelabs.com/) or [Applitools](https://applitools.com/))
+- support of functional as well as non function testing aspects (e.g. visual regression or frontend performance testing)
+- debugging browser in an automated fashion (e.g. using the [Chrome DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/))
+- scrapping of websites and other web/mobile interfaces
+- providing technical content and best practices in form of [boilerplates](https://webdriver.io/) and [blog posts](https://webdriver.io/blog/)
+- providing technical help in form of a [community channel](https://gitter.im/webdriverio/webdriverio)
+
+### 1.2: Out-of-Scope
+
+- API testing
+- Load testing
+
+## Section 2: Relationship with OpenJS Foundation CPC.
+
+Most large, complex open source communities have both a business and a technical governance model. Technical leadership for the projects within the OpenJS Foundation is delegated to the projects through their project charters by the OpenJS Cross Project Council (CPC). In the case of the WebdriverIO project, it is delegated to the WebdriverIO Technical Steering Committee (“TSC”). OpenJS Foundation’s business leadership is the Board of Directors (the “Board”).
+
+This Technical Steering Committee Charter reflects a carefully constructed balanced role for the TSC and the CPC in the governance of the OpenJS Foundation. The charter amendment process is for the TSC to propose changes using simple majority of the full TSC, the proposed changes being subject to review and approval by the CPC. The CPC may additionally make amendments to the TSC charter at any time, though the CPC will not interfere with day-to-day discussions, votes or meetings of the TSC.
```

Comment:
This Technical Steering Committee Charter reflects a carefully constructed and balanced role for the TSC and the CPC in the governance of the OpenJS Foundation.